web/packages/dmt-core/src/components/Header.tsx

```diff
@@ -103,6 +104,7 @@ export const Header = (props: {
 }): JSX.Element => {
   const { appName, urlPath, allApps } = props
   const [version, setVersion] = useState<string>('Version not loaded')
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore